src/compas_robots/model/robot.py

```diff
@@ -990,6 +1004,181 @@ def _check_link_name(self, name):
         if name in all_link_names:
             raise ValueError("Link name '%s' already used in chain." % name)
 
+    # --------------------------------------------------------------------------
+    # Methods for accessing the visual and collision geometry
+    # --------------------------------------------------------------------------
+
+    def _extract_link_meshes(self, link_elements, meshes_at_link_origin=True):
+        # type: (List[Visual] | List[Collision], Optional[bool]) -> List[Mesh]
+        """Extracts the list of compas meshes from a link's Visual or Collision elements.
+
+        Parameters
+        ----------
+        link_elements : list of :class:`~compas_robots.model.Visual` or :class:`~compas_robots.model.Collision`
+            The list of Visual or Collision elements of a link.
+        meshes_at_link_origin : bool, optional
```

Comment:
why would one not want to transform?

Reply:
So this is probably for completeness sake, because technically, if someone wants to get the original mesh (of the robot model) and save it to another format, and does not want to change the URDF origin of that link, that mesh should not be transformed. 

So URDF give this flexibility to model a mesh not at the origin of the link, and some robots models actually do that. (yes, crazy) And we respect that. For out current and immediate application, which is to visualize the robot and to repackage and send it to PyBullet, all those application, we want the transformed mesh, makes downstream stuff simplier. So yes, there are reasons for both options to exist.